packages/packages.go

```diff
@@ -81,9 +81,15 @@ type Packages struct {
 
 // PkgInfo describes a package.
 type PkgInfo struct {
-	Name, Arch, Version string
+	Name, Version string
+	Arch          OSArch
+	Source        Source
+}
 
-	Source Source
+// OSArch describes a OS Architecture specified for package.
+type OSArch struct {
```

Comment:
I suggest to create constructor function like

```
func NewArchitecture(raw string) Architecture {
    return Architrecture{StandardArch: osinfo.Architecture(raw), RawArch: raw}
}
```

That would create all code that initialize structure cleaner